pyproject.toml

```diff
@@ -0,0 +1,7 @@
+[tool.towncrier]
```

Comment:
There's a reasonable chance the presence of this file will break #955. @themperek, could you test with an empty `pyproject.toml` file on your branch and see if things are ok still?

Reply:
It seems to not cause any issues.